app/controllers/step_by_step_pages_controller.rb

```diff
@@ -90,7 +90,7 @@ def schedule
         render :schedule
       elsif @step_by_step_page.update_attributes(scheduled_at: scheduled_at)
         schedule_to_publish
-        note_description = "Minor update scheduled by #{current_user.name} for publishing at #{scheduled_at}"
+        note_description = "Minor update scheduled by #{current_user.name} for publishing at #{format_full_date_and_time(scheduled_at)}"
```

Comment:
Nitpick: I think the note description should say "on" rather than "at", e.g. "on Saturday, 20 April 2030 at 10:26 am"

Reply:
Agreed. Good spot.